cluster-service/Makefile

```diff
@@ -19,14 +19,19 @@ deploy:
 			-g ${RESOURCEGROUP} \
 			-n clusters-service \
 			--query clientId) && \
+	AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID=$(shell az ad app list \
+			--display-name aro-hcp-dev-first-party \
+			--query '[*]'.appId \
+			-o tsv) && \ 
 	CS_SERVICE_PRINCIPAL_CREDS_BASE64='$(shell az keyvault secret show --vault-name "service-kv-aro-hcp-dev" --name "aro-hcp-dev-sp-cs" | jq .value -r | base64 | tr -d '\n')' && \
 	oc process --local -f deploy/openshift-templates/arohcp-service-template.yml \
 	  -p AZURE_CS_MI_CLIENT_ID=$${AZURE_CS_MI_CLIENT_ID} \
 	  -p REGION=${REGION} \
 	  -p CS_SERVICE_PRINCIPAL_CREDS_BASE64=$${CS_SERVICE_PRINCIPAL_CREDS_BASE64} \
 	  -p IMAGE_REGISTRY=${ARO_HCP_IMAGE_ACR}.azurecr.io \
 	  -p IMAGE_REPOSITORY=app-sre/uhc-clusters-service \
-	  -p IMAGE_TAG=2256c75 | oc apply -f -
+	  -p AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID=$${AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID} \
+	  -p IMAGE_TAG=3482df90 | oc apply -f -
```

Comment:
double checked the tag, looks like its 7 chars long. It should be `3482df9`

You can check it like this i believe:
```az acr repository show -n arohcpdev --image app-sre/uhc-clusters-service:3482df9```

and to check the available tags in that repo: 
```az acr repository show-tags -n arohcpdev --repository app-sre/uhc-clusters-service --detail --orderby time_desc --top 5```

Reply:
> double checked the tag, looks like its 7 chars long. It should be `3482df9`
> 
> You can check it like this i believe: `az acr repository show -n arohcpdev --image app-sre/uhc-clusters-service:3482df9`
> 
> and to check the available tags in that repo: `az acr repository show-tags -n arohcpdev --repository app-sre/uhc-clusters-service --detail --orderby time_desc --top 5`

changed, thanks for pointing it out